notifications/src/main/java/org/openedx/notifications/domain/interactor/NotificationsInteractor.kt

```diff
@@ -40,4 +49,56 @@ class NotificationsInteractor(
     suspend fun markAllNotificationsAsRead(): Boolean {
         return repository.markNotificationAsRead(notificationId = null)
     }
+
+    fun shouldShowNotificationsPrimer(): Boolean {
```

Comment:
Please replace the hard coded values with the const value, including dismissal & number of days